pkg/gofr/handler.go

```diff
@@ -101,14 +91,21 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			err = gofrHTTP.ErrorRequestTimeout{}
 		}
 	case <-done:
-		if websocket.IsWebSocketUpgrade(r) {
-			// Do not respond with HTTP headers since this is a WebSocket request
-			return
-		}
+		handleWebSocketUpgrade(r)
 	case <-panicked:
 		err = gofrHTTP.ErrorPanicRecovery{}
 	}
 
+	// **Handle Custom Headers** if `result` is a `Response`.
```

Comment:
I'm unsure about the formatting here

In a godoc, I would have understand

Here I don't get what it brings.

I might not have raised this, but I also face a misunderstanding about the fact I don't get why there is a need to use emphasis on this

I could have understood if it was


```suggestion
	// handle **Custom Headers** if `result` is a `Response`.
```

But then, I would be lost anyway. Why would we need to emphasis on the fact it's custom headers, maybe you could explain

And finally, why using emphasis and not simply use quote.

```suggestion
	// handle "Custom Headers" if `result` is a `Response`.
```

Thanks for reading this almost useless feedback about a comment 😅😬😂